docs/source/documentation/data-products/components/product-specification.html.md.erb

```diff
@@ -15,9 +22,18 @@ You must also provide contact information for the product.
 ```yaml
 name: example_data_product
 description: Example Data Product contains published prison population from 2001 to present
-status: string (enum)
+status: production
+email: data.product.contact@justice.gov.uk
 ```
 
+including optional fields:
+```yaml
+name: example_data_product
+description: Example Data Product contains published prison population from 2001 to present
+status: production
+email: data.product.contact@justice.gov.uk
+tags: {"Sandbox": "True"}
```

Comment:
I have a question these docs should probably clarify: is there a predefined list of tags you can use, or does the data owner have free reign to come up with their own tagging conventions?

I ask because OpenMetadata's concept of a tag forces you to predefine "classifications" before you can use them, and there are a few that already exist, e.g. for specifying environments etc. If we don't have a fixed list of valid tag names, then data owners will diverge and the tags won't be very useful to consumers who are browsing via the catalogue and want to filter in some way.

We don't have to implement our tags using OpenMetadata tags, (we could use custom fields) but if we want to take advantage of it in the UI then implementing it their way would be simplest.

Reply:
I had this discussion with @jemnery - we settled on user-specified tags (free text), though this was not considering a link with OpenMetadata's tags. 

Can we link existing free-text tags with OpenMetadata and then have non-existing tags fall into a custom field? 
E.g. 
- if all user-specified tags exist in OpenMetadata, they all match to the OpenMetadata tags
- if no user-specified tags exist in OpenMetadata, they all go into a custom field
- if some user-specified tags exist in OpenMetadata, the matching tags become OpenMetadata tags, the others go into a custom field

Reply:
I've moved this proposal to https://github.com/ministryofjustice/data-platform/issues/2249